src/ripple/test/jtx/Env.h

```diff
@@ -388,6 +388,13 @@ class Env
     void
     submit (JTx const& jt);
 
+    /** Sign-and-submit an existing, but
+        possibly incomplete JTx.
+        This calls postconditions.
+    */
```

Comment:
Contrast with the Javadoc on `Env::submit`. 
```
    /** Submit an existing JTx.
        This calls postconditions.
    */
```
I wanted this one to be distinct enough that no one has to ask "Why would I use `sign_and_submit` over `submit`?" The name is a big hint, of course. Heh.

Anyway, I'm taking your suggestion, but I suspect this isn't the last update.